src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -116,6 +116,7 @@ struct sd_dhcp_client {
         sd_event_source *timeout_t1;
         sd_event_source *timeout_t2;
         sd_event_source *timeout_expire;
+        sd_event_source *timeout_ipv6_only_preferred;
```

Comment:
`_preferred` is an option name, and corresponds to the timespan, not a timer. Please name `timeout_ipv6_only_wait` or so, as RFC call the timer `V6ONLY_WAIT`.